### ISSUE TITLE
Keyboard Shortcut: Space to start / stop playback

### ISSUE THREAD

**Issue by timpulver** [quodlibet/quodlibet#1701]
Using the space key to start / stop music playback is a universal thing. Almost all native players support it as well as websites like Youtube. Would be great if Quod Libet would adapt to the standards. Seems like Quod Libet only behaves *normal* when the focus is on the play button. In list view pressing space results in the selected track being played.

I’m on Quod Libet 3.5.0 on Mac OS X 10.10.5

**Comment by timpulver** [comment]
Currently there is no shortcut to play / pause, right?
At least it does not show up in the [docs](https://quodlibet.readthedocs.org/en/quodlibet-3.5/guide/shortcuts.html).

**Comment by lazka** [comment]
The docs only show shortcuts that aren't shown in the menus (See "Control" in the main menu). play/pause can be toggled with Command+Space on OSX

**Comment by timpulver** [comment]
Thanks, but `⌘+SPACE` is a universal Mac shortcut for the spotlight search. So it cannot be used.
Btw. `⌘+;` is normally also a standard-shortcut (to open the preferences of the current application).

**Comment by lazka** [comment]
OK, that needs to change then, thanks.

**Comment by timpulver** [comment]
Another shortcut which feels a bit strange is next track / seek forward respectively previous track / seek backwards. It’s not intuitive because you have to use another modifier key (⌥ / ⌘).

Most intuitive would be for me:

`⌘  + →` seek forward
`⌥ + ⌘  + →` next track
`⌘  + ←` seek backwards
`⌥ + ⌘  + ←` next track

**Comment by urielz** [comment]
@timpulver small correction: it's ⌘+, to open the preferences.

Table 68-3 lists the standard keyboard shortcuts in OS X:

https://developer.apple.com/library/mac/documentation/UserExperience/Conceptual/OSXHIGuidelines/Keyboard.html#//apple_ref/doc/uid/20000957-CH84-SW4

Another useful addition would be ⌘+` to cycle through the open browsers.

**Comment by timpulver** [comment]
@urielz You’re right!
Good to have an eye on the other ones!

**Comment by ochurlaud** [comment]
@lazka: would it be possible that the <space> does pause/play pause eventually? It's often the case on music players, but here it restarts the track list.

Maybe you could let us choose? (and allow the user to change the shortcuts like they want?). A CTRL+ALT+M to put a random album, SPACE to pause, and so on.... (if it's another issue, say it I'll open a new one)

**Comment by NoahAndrews** [comment]
Yes! It's so annoying to expect the music to pause and instead have it play the first song. Please change that :)

**Comment by CreamyCookie** [comment]
The "Space to play" issue is already mentioned in #1288. I'm renaming this to focus on the Mac aspects.

**Comment by paxter** [comment]
Anything new here?
